assets/airports/kbos.json

```diff
@@ -1,713 +1,8666 @@
+      "rwy": {
+        "4R":  [["TTERI", "A140|S250"], ["WAATR", "A100"], ["FLUTI", "A90"], ["GRIFI", "A70"], ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
+        "4L":  [["TTERI", "A140|S250"], ["WAATR", "A100"], ["FLUTI", "A90"], ["GRIFI", "A70"], ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
+        "15R": [["GRGIO", "A90"], ["BRGIT", "A70"], ["HNOVR", "A50|S220"], ["RDHOK", "A50|S210"], ["TKMAN", "A50|S210"]],
+        "22R": [["GRGIO", "A90"], ["BRGIT", "A70"], ["ADDDA", "A60|S220"], ["LEEZI", "A60|S210"]],
+        "22L": [["GRGIO", "A90"], ["BRGIT", "A70"], ["ADDDA", "A60|S220"], ["LEEZI", "A60|S210"]],
+        "27":  [["PUDJJ", "A90"], ["AYBEE", "A60|S210"]],
+        "33L": [["TTERI", "A110|S250"], ["WAATR", "A70"], ["PLGRM", "A60"], ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]],
+        "32":  [["TTERI", "A110|S250"], ["WAATR", "A70"], ["PLGRM", "A60"], ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]]
+      }
+    },
+    "ROBUC2": {
+      "icao": "ROBUC2",
+      "name": "Roebook Two",
+      "entryPoints": {
+        "JFK":   ["JFK", ["NEWES", "A330-"], ["FEXXX", "A230"]],
+        "MERIT": ["MERIT"]
+      },
+      "body": [["RUIZE", "A230|S350"], ["AWLIN", "A220|S320"], ["BANKI", "A210|S300"], ["ROBUC", "A180|S260"], ["PROVI", "A160"], ["JOODY", "A140"]],
+      "rwy": {
+        "4R":  [["SOFEE", "A100|S220"], ["ERNEI", "A80"], ["GOSHI", "A60|S210"]],
+        "4L":  [["SOFEE", "A100|S220"], ["ERNEI", "A80"], ["GOSHI", "A60|S210"]], 
+        "15R": [["JAYNA", "A110|S250"], ["BOXRS", "A80|S250"], ["CHERS", "A60|S220"], ["RDHOK", "A60|S210"], ["TKMAN", "A60|S210"]],
+        "22R": [["KRANN", "A120|S250"], ["CRADL", "A100"], ["KLEBB", "A80|S250"], ["HOKDU", "A50|S210"]],
+        "22L": [["KRANN", "A120|S250"], ["CRADL", "A100"], ["KLEBB", "A80|S250"], ["HOKDU", "A50|S210"]],
+        "27":  [["KRANN", "A120|S250"], ["CRADL", "A100"], ["KLEBB", "A80|S250"], ["HOKDU", "A50|S210"]],
+        "33L": [["JAYNA", "A110|S250"], ["ANSLY", "A80|S240"], ["BEREI", "A60|S220"], ["BBOGG", "A60|S210"]],
+        "32":  [["JAYNA", "A110|S250"], ["ANSLY", "A80|S240"], ["BEREI", "A60|S220"], ["BBOGG", "A60|S210"]]
+      }
+    },
+    "WOONS2": {
+      "icao": "WOONS2",
+      "name": "Woons Two",
+      "entryPoints": {
+        "ORW": ["ORW"]
+      },
+      "body": ["JEWIT", "FOSTY", "HEFTY", "WOONS"],
+      "rwy": {
+        "4R":  [],
+        "4L":  [],
+        "15R": [],
+        "22R": [],
+        "22L": [],
+        "27":  [],
+        "33L": [],
+        "32":  []
+      }
+    }
+  },
+  "spawnPatterns": [
+    {
+      "origin": "kbos",
+      "destination": "",
+      "category": "departure",
+      "route": "KBOS.BLZZR3.BLZZR",
+      "altitude": "",
+      "speed": "",
+      "method": "random",
+      "rate": 3,
+      "airlines": [
+        ["aal",     10],
+        ["aal/long", 4],
+        ["asq",      2],
+        ["dal",      7],
+        ["dal/long", 3],
```

Comment:
Single spacing only, except in the definitions of non-underscore-prefixed fixes.